cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -2280,6 +2304,8 @@ describe('CourseOutlinePage', function() {
                     createMockSectionJSON({}, [createMockSubsectionJSON()])
                 );
 
+                expect($('.outline-subsection .status-grading-date')).not.toExist();
```

Comment:
What is grading date? The instructor paced version?

Reply:
Yup! But I'll take it out since it doesn't make sense for self paced courses to have them in the first place!